frontend/src/scenes/plugins/plugin/PluginCard.tsx

```diff
@@ -214,6 +217,19 @@ export function PluginCard({
                                 />
                             ) : pluginId ? (
                                 <>
+                                    {shouldShowAppMetrics && pluginConfig?.id ? (
```

Comment:
we should also likely hide the page, currently it seems we just gate showing the button, but anyone knowing what the link would be can access it